File: vpn-control-panel-backend/src/main/java/io/erisdev/vpncontrolpanelbackend/model/ClientStatus.java
package io.erisdev.vpncontrolpanelbackend.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ClientStatus {
    @JsonProperty("cn")  private String cn;
    @JsonProperty("real_ip") private String realIp;
    @JsonProperty("vpn_ip")  private String vpnIp;
    @JsonProperty("bytes_in")  private long bytesIn;
    @JsonProperty("bytes_out") private long bytesOut;
    @JsonProperty("connected_since_epoch") private long connectedSinceEpoch;
    @JsonProperty("connected_since") private String connectedSince;
    private String status;

}

File: vpn-control-panel-frontend/src/app/model/client-status.ts
export class ClientStatus {
  cn: string = '';
  realIp: string = '';
  vpnIp: string = '';
  bytesIn: number = 0;
  bytesOut: number = 0;
  connectedSince: Date = new Date();
  status: string = '';
  lastSeen: Date = new Date();
}
